uefi/src/runtime.rs

```diff
@@ -270,8 +271,7 @@ impl VariableKeys {
     fn new() -> Self {
         // Create a the name buffer with a reasonable default capacity, and
         // initialize it to an empty null-terminated string.
```

Comment:
Update this comment, we're no longer just creating initial capacity, we're initializing to 512 bytes. Also add some details about why we need to start with a large buffer, and the name of the model you observed the bad behavior on.